dimod/sampleset.py

```diff
@@ -1083,6 +1083,55 @@ def append_variables(self, samples_like, sort_labels=True):
                                        sort_labels=sort_labels,
                                        **self.data_vectors)
 
+    def lowest(self, rtol=1.e-5, atol=1.e-8):
+        """Return a SampleSet containing the samples with the lowest energy.
+
+        Args:
+            rtol (float, optional, default=1.e-5):
+                The relative tolerance. See :func:`numpy.isclose`.
+
+            atol (float, optional, default=1.e-8):
+                The absolute tolerance. See :func:`numpy.isclose`.
+
+        Returns:
+            :obj:`.SampleSet`: A new sample set containing the samples with an
+            energy within tolerance of the minimum energy of the sample set.
+
+        Examples:
+            >>> sampleset = dimod.ExactSolver().sample_ising({'a': .001},
+            ...                                              {('a', 'b'): -1})
+            >>> print(sampleset.lowest())
+               a  b energy num_oc.
+            0 -1 -1 -1.001       1
+            ['SPIN', 1 rows, 1 samples, 2 variables]
+            >>> print(sampleset.lowest(atol=.1))
+               a  b energy num_oc.
+            0 -1 -1 -1.001       1
+            1 +1 +1 -0.999       1
+            ['SPIN', 2 rows, 2 samples, 2 variables]
+
+        Note:
+            The energy is relative to the lowest energy in the sample set. This
```

Comment:
This statement needs to be referenced to the tolerance parameters.
"Tolerance parameters are relative the the lowest energy in the SampleSet, not the ground-state energy of the binary quadratic model.